pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -650,24 +650,13 @@ async def verify_attached_operators(self, *advisories: List[int]):
         await exectools.cmd_assert_async(cmd, env=self._elliott_env_vars, cwd=self.working_dir)
 
     @retry(reraise=True, stop=stop_after_attempt(3), wait=wait_fixed(10))
-    def send_live_id_request_mail(self, advisories: Dict[str, int]):
-        subject = f"Live IDs for {self.release_name}"
-        main_advisory = "image" if self.release_version[0] >= 4 else "rpm"
-        content = f"""Hello docs team,
-
-ART would like to request Live IDs for our {self.release_name} advisories:
-{main_advisory}: https://errata.devel.redhat.com/advisory/{advisories[main_advisory]}
-
-This is the current set of advisories we intend to ship:
-"""
+    async def create_live_ids(self, advisories: Dict[str, int]):
         for kind, advisory in advisories.items():
-            content += (
-                f"- {kind}: https://errata.devel.redhat.com/advisory/{advisory}\n"
+            _LOGGER.info(f"Giving {kind} advisory {advisory} a Live ID.")
+            await exectools.cmd_assert_async(
```

Comment:
How does this return when a LiveID already has been assigned?

In order to have meaningful return codes, we might need to read if a LiveID has been assigned already, like this: https://github.com/openshift/aos-cd-jobs/blob/76d7ee16bb34dd81eb284ac5aaac53cd1ca9533b/pipeline-scripts/release.groovy#L120

Reply:
it turns out to be idempotent AFAICS. checking the return code and maybe retrying would probably be a good idea though (and a slack msg if it fails - it's not critical). if it's any more complicated than this command (or even if that's all it is), might make a good elliott command

Reply:
I think we only need to request the live ID just after a new advisory is created. When updating an existing advisory, we don't need to request again.

Reply:
So this function can be chained after the create advisory logic.

Reply:
yeah this should just go in `elliott create`